apps/spruce/cypress/integration/version/version_timing.ts

```diff
@@ -0,0 +1,174 @@
+import { clickOnPageSizeBtnAndAssertURLandTableSize } from "../../utils";
+
+describe("Version Timing Tab without a variant selected", () => {
+  beforeEach(() => {
+    cy.visit("/version/5e4ff3abe3c3317e352062e4/version-timing");
+  });
+  it("shows a chart of all variants in the version", () => {
+    const expectedVariants = ["Ubuntu 16.04", "Lint", "Race Detector"];
+    cy.get("svg > g > text").then(($items) => {
+      const textFound = Array.from($items, (item) => item.innerHTML);
+      expectedVariants.forEach((variant) => {
+        expect(textFound).to.include(variant);
+      });
+    });
+  });
+  it("allows the user to select a variant and see the tasks in it", () => {
```

Comment:
The test doesn't assert the task visibility. You can change the test name or assert that the tasks are visible.